packages/salesforcedx-vscode-apex/README.md

```diff
@@ -20,18 +20,18 @@ To see code-completion suggestions, press Ctrl+space when you’re working in a
 ## View or Jump to Definitions
 You can preview, view, or go to definitions of:  
 * Apex  
+  * classes (from within extending classes)  
```

Comment:
It only works for the usage site in "extends x" or "implements xx, yy".  It doesn't work (yet) for Parent foo; where you click on Parent.   That work is on future work items.   Not quite sure how to say this in the docs.   We decided it was better to release the functionality that we have rather than wait until all possible sites are working